content/discord.md

```diff
@@ -0,0 +1,14 @@
+@title discord
+@navbar
+
+# Polympics discord server
+
+https://discord.gg/kJG7eM4hDG
```

Comment:
```suggestion
## [Polympics Discord Server](https://discord.gg/kJG7eM4hDG)

The official home of the event. Join to start chatting with fellow participants and team members!
```

Reply:
Note that there should never be more that one single-`#` on a page, Google will rank pages lower for it.